extensions/mongoProcessor/MongoQueueProcessor.js

```diff
@@ -90,12 +137,26 @@ class MongoQueueProcessor {
             });
             this._consumer.on('ready', () => {
                 consumerReady = true;
-                this._consumer.subscribe();
+                const paused = options && options.paused;
+                this._consumer.subscribe(paused);
                 this.logger.info('mongo queue processor is ready');
             });
         });
     }
 
+    getServiceVars() {
```

Comment:
I'm okay with this naming, however across backbeat the method for this purpose is consistently called `getStateVars`. It could prove helpful over time to use that convention.

Reply:
I wanted to keep this as a separate method because it requires anyone using the mixin to create the initial methods for set up (i.e. `getServiceVars` and `_getZkSiteNode`) and also separates concerns between what `getStateVars` is used for vs what `getServiceVars` is used for.

I think the naming can be improved to clarify and make it more apparent. Maybe I can call this something like `getPauseResumeVars`? And I can add a comment at top of the mixin file to specify necessary methods/variables for use.